packages/web-react/src/components/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -56,10 +56,14 @@ describe('TextArea', () => {
     expect(element).toHaveClass('TextArea--fluid');
   });
 
-  it.each([['success'], ['warning'], ['error']])('should have %s classname', (validationState) => {
-    const dom = render(<TextArea id="textarea" label="Label" validationState={validationState as ValidationState} />);
+  // @deprecated Will be removed in next major version.
+  it.each([ValidationStates.SUCCESS, ValidationStates.WARNING, ValidationStates.DANGER, ['error']])(
```

Comment:
`Object.values` or something else to get all values from `ValidationStates`?

Reply:
E.g. validation state props test...

Reply:
fixed & done